util/selection/selection_test.go

```diff
@@ -58,37 +48,37 @@ func init() {
 	globalCase50 = randomTestCase(50)
 }
 
-func (s *testSuite) TestSelection(c *C) {
+func TestSelection(t *testing.T) {
```

Comment:
It seems that those test cases could be paralleled.

Reply:
thanks, fixed and updated.